internal/tree/tree.go

```diff
@@ -0,0 +1,48 @@
+package tree
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+func tab(tabs uint) {
+	for i := 0; i < int(tabs); i++ {
+		fmt.Print("│   ")
+	}
+}
+
+func Tree(path string, depthLevelFlag, numOfTabs uint, dirCount, fileCount *uint) error {
+	contents, err := os.ReadDir(path)
+	if err != nil {
+		return fmt.Errorf("%s: [error opening dir]", path)
+	}
+
+	for _, content := range contents {
+		if strings.HasPrefix(content.Name(), ".") {
```

Comment:
why not print files starting with "."?

Reply:
Same behaviour as the actual `tree` command.
Should I print dot files or keep it as it is?